src/index.html

```diff
@@ -9,6 +9,7 @@
   <link rel="icon" type="image/x-icon" href="assets/icons/Logo.png">
   <link rel="manifest" href="manifest.json">
   <link href="https://fonts.googleapis.com/css?family=Sunflower:300" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/pretty-checkbox@3.0/dist/pretty-checkbox.min.css"/>
```

Comment:
I'm slightly CDN-averse. I'd be tempted to move this into package.json and import it in App.css.

Reply:
I agree. I wish we could use the uncompiled form, but unfortunately this one was written in SCSS